### PR TITLE
Docs for RenderSliverFixedExtentList

### PR DESCRIPTION
**Pull request by abarth**
Also, some renames for more accuracy.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie @HansMuller

File: packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart [hunk 1]
```diff
@@ -10,27 +10,77 @@ import 'box.dart';
 import 'sliver.dart';
 import 'sliver_multi_box_adaptor.dart';
 
+/// A sliver that contains multiple box children that have the same extent in
+/// the main axis.
+///
+/// [RenderSliverFixedExtentBoxAdaptor] places its children in a linear array
+/// along the main axis. Each child is forced to have the [itemExtent] in the
+/// main axis and the [SliverConstraints.crossAxisExtent] in the cross axis.
```
Comment (Hixie) [inline packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart]: You should mention here that `itemExtent` isn't just a field here but is something you override in a subclass.

File: packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart [hunk 2]
```diff
@@ -184,6 +271,11 @@ class RenderSliverFill extends RenderSliverFixedExtentBoxAdaptor {
   @override
   double get itemExtent => constraints.viewportMainAxisExtent * viewportFraction;
 
+  /// The fraction of the viewport that each child should fill in the main axis.
+  ///
+  /// If this fraction is less than 1.0, more than one child will be visibile at
```
Comment (Hixie) [inline packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart]: visible

**Comment by Hixie** [comment]
LGTM

**Pull request merged**